repose-aggregator/components/filters/uri-stripper-filter/src/main/resources/META-INF/schema/config/uri-stripper.xsd

```diff
@@ -65,14 +71,160 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="LinkResource">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    A specific ReST resource that contains one or more links which should have the stripped URI
+                    path segment re-inserted.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:all>
+            <xs:element name="request" type="HttpMessage" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="response" type="HttpMessage" minOccurs="0" maxOccurs="1"/>
+        </xs:all>
```

Comment:
So my take is, i think about things as a resource. I don't think of all my requests, and then all my responses. I think about the fact that i have this resource which has this input and this output, and this config aligns with that.